sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-processing-steps/validate-disbursement-base.ts

```diff
@@ -52,6 +54,19 @@ export abstract class ValidateDisbursementBase {
         ECertFailedValidation.DisabilityStatusNotConfirmed,
       );
     }
+    // No estimated awards amounts to be disbursed.
+    const totalEstimatedAwards =
+      eCertDisbursement.disbursement.disbursementValues.reduce(
```

Comment:
The disbursementValues can be null/undefined if no values present for the disbursement.

Reply:
Typeorm will return an empty array and there is an E2E test checking this scenario.